tests/tests_suite_test.go

```diff
@@ -57,6 +36,32 @@ type App struct {
 	URL  string
 }
 
+type TestData struct {
+	Username      string
+	Password      string
+	Email         string
+	KeyName       string
+	KeyPath       string
+	Profile       string
+	ProfilePath   string
+	ControllerURL string
+}
+
+var adminTestData TestData
+var testRoot, testHome, keyPath, gitSSH string
```

Comment:
I can't seem to find where `testHome` is actually set -- could you point me to the spot?